gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Channelizer.java

```diff
@@ -180,8 +182,9 @@ public void configure(final ChannelPipeline pipeline) {
             final int maxContentLength = cluster.connectionPoolSettings().maxContentLength;
             handler = new WebSocketClientHandler(
                     new WebSocketClientHandler.InterceptedWebSocketClientHandshaker13(
-                            connection.getUri(), WebSocketVersion.V13, null,true,
-                            EmptyHttpHeaders.INSTANCE, maxContentLength, true, false, -1,
+                            connection.getUri(), WebSocketVersion.V13, null, true,
+                            new DefaultHttpHeaders().set(EmptyHttpHeaders.INSTANCE).set("user_agent", UserAgent.getUserAgent()),
```

Comment:
Why do we need `.set(EmptyHttpHeaders.INSTANCE)` ?

Reply:
It is not needed. I will remove it.